features/netsocket/OnboardNetworkStack.h

```diff
@@ -103,7 +108,7 @@ class OnboardNetworkStack : public NetworkStack {
          * @param    buflen     size of supplied buffer
          * @return              Pointer to a buffer, or NULL if the buffer is too small
          */
-        virtual char *get_ip_address(char *buf, nsapi_size_t buflen) = 0;
+        virtual char *get_ip_address(char *buf, nsapi_size_t buflen, const char *interface_name) = 0;
```

Comment:
Should default to to = NULL, parameter description missing

Reply:
Done